cpp/scripts/include_checker.py

```diff
@@ -30,6 +33,8 @@ def parse_args():
     argparser.add_argument("--regex", type=str,
                            default=r"[.](cu|cuh|h|hpp|hxx|cpp)$",
                            help="Regex string to filter in sources")
+    argparser.add_argument("--inplace", action="store_true", required=False,
```

Comment:
THanks @MatthiasKohl for automating this process! Can you add a few lines to our cpp dev guide about this check and how to "fix" the errors by using this `--inplace` option? Relevant section that needs this update is here: https://github.com/rapidsai/cuml/blob/branch-0.14/wiki/cpp/DEVELOPER_GUIDE.md#coding-style

Reply:
Makes sense, will do!